src/styles/blocks/phone-card.scss

```
@@ -0,0 +1,102 @@
+@use "../utils/extends";
+@use "../utils/variables" as *;
+@use "../utils/function";
+
+.card-container {
+  max-width: function.rowWidth($card-width, $card-offset, $cards-per-row);
+  @extend %centre-flex;
+  flex-grow: 1;
+  flex-wrap: wrap;
+}
+
+.phone-card {
+  @extend %centre-flex;
+  width: #{$card-width};
+  margin: 50px 24px 46px;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: #{$card-padding};
```

just this way, make sure to fix it all over
```suggestion
  padding: $card-padding;
```